### DISCUSSION TITLE
Integrate with the Sleeve app

### DISCUSSION THREAD

**Discussion by double-u-a** [SubmarinerApp/Submariner#195] [Ideas]
There's a really nice app called [sleeve](https://replay.software/sleeve) which puts the album cover and controls on the desktop, they are open to integrations with other apps, no idea about compatibility but it looks like a simple AppleScript implementation, the integration instructions are below. Would also mean Sleeve users learn about Submariner and subsonic servers.

https://replay.software/help/sleeve/developers

**Comment by NattyNarwhal** [comment]
Looks like a matter of adding AppleScript support per here: https://stackoverflow.com/a/10773994

**Comment by NattyNarwhal** [comment]
I've implemented the bare minimum of what looks to be required per their documentation. I'll try to poke them later.

**Comment by NattyNarwhal** [comment]
The Sleeve developers say they aren't going to support it right now, but they might have some kind possible path forward in the future.
